core/profiling/profiling.h

```diff
@@ -46,12 +45,25 @@
 #if defined(GODOT_USE_TRACY)
 // Use the tracy profiler.
 
+#include "core/string/ustring.h"
+
 #define TRACY_ENABLE
+
 #include <tracy/Tracy.hpp>
 
+inline const char *intern_string(const char *p_name) {
+	return p_name;
+}
+const char *intern_string(const String &p_name);
+const char *intern_string(const CharString &p_name);
```

Comment:
These should get a namespace, like `Internal`. Otherwise, someone might use them by accident.